icons/keyboard.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect x="2" y="4" width="20" height="16" rx="2" ry="2" />
+  <path d="M6 8h.001m4 0h.001m4 0h.001m4 0h.001M8 12h.001m4 0h.001m4 0h.001M7 16h10" />
```

Comment:
Can you maybe split this up in separate parts, some design programs don't like this combined paths.
If you need any help with that let me know.